src/pages/LandingPage/LandingPage.tsx

```diff
@@ -6,13 +6,14 @@ import LoadableContent from 'src/components/LoadableContent';
 import Loader from 'src/components/Loader';
 import { Heading } from './styled';
 import { fetchEvents } from 'src/api/events';
+import { subHours } from 'date-fns';
 
 function LandingPage() {
   return (
     <section id="landing-page">
       <Heading>Events near you</Heading>
       <LoadableContent
-        resolveContent={fetchEvents}
+        resolveContent={() => fetchEvents(subHours(new Date(), 12))}
```

Comment:
Instead of filtering events based on the start date, I think it is better to filter based on the end date. This is because we probably want ongoing events to still show up in the list.